packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -123,17 +134,17 @@ class ButtonGroup extends React.Component {
 
     const allButtons = children ? [] : undefined;
     // eslint-disable-next-line no-nested-ternary
-    const btnRole = onChange ? isMultiSelect ? 'checkbox' : 'radio' : 'button';
+    const btnRole = onChange ? isMultiSelect ? 'checkbox' : 'radio' : '';
 
     React.Children.forEach(children, (child, index) => {
       const isSelected = selectedKeys.indexOf(child.key) > -1;
       const cloneChild = React.cloneElement(child, {
-        role: btnRole,
+        [btnRole !== '' && 'role']: btnRole,
```

Comment:
why was this change made ??

Reply:
Because of role button it is reading as null.

Reply:
So are you just trying to only create the role btnRole variable has a value?  I also am not clear on why you are trying to accomplish here.  You also mentioned JAWS on Edge.  Do we know if there was any impact to VoiceOver or JAWS on other browsers by this change?  Could you not just use something like
 role: btnRole || undefined

Reply:
I have made changes as per Suggestion.